src/main/java/org/springframework/data/couchbase/core/ExecutableRemoveByIdOperation.java

```diff
@@ -50,6 +50,11 @@
 
 	}
 
-	interface ExecutableRemoveById extends RemoveByIdWithDurability {}
+	interface RemoveByIdWithCas extends RemoveByIdWithDurability {
+
+		RemoveByIdWithDurability withCas(Long cas);
```

Comment:
why is this a boxed long and not an unboxed `long`?

Reply:
So that null can have different semantics than zero.